h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -74,6 +75,58 @@ public void initActualParamValues() {
     //the training frame used for blending (from which predictions columns are computed)
     public Key<Frame> _blending;
 
+    public enum MetalearnerTransform {
+      NONE(null),
+      Logit(new MRTask() {
+        @Override
+        public void map(Chunk[] cs) {
+          double p, logit;
+          for (int c = 0; c < cs.length; c++) {
+            for (int i = 0; i < cs[c]._len; i++) {
+              p = cs[c].atd(i);
+              logit = Math.log(p/(1-p));
```

Comment:
you can either use existing LogitFunction or just use LogExpUtil.log that prevents issues

Reply:
Thanks for the tip!